source/magnifier/__init__.py

```diff
@@ -0,0 +1,74 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2025 NV Access Limited, Antoine Haffreingue
+# This file may be used under the terms of the GNU General Public License, version 2 or later, as modified by the NVDA license.
+# For full terms and any additional permissions, see the NVDA license file: https://github.com/nvaccess/nvda/blob/master/copying.txt
+
+"""
+NVDA Magnifier module.
+Handles module initialization, configuration and settings interaction.
+"""
+
+import config
+
+from .fullscreenMagnifier import FullScreenMagnifier
+
+_magnifier: FullScreenMagnifier | None = None
+
+# Configuration specification
+confspec = {
+    "magnifier": {
+        "defaultZoomLevel": "float(min=1.0, max=10.0, default=2.0)",
+        "defaultFullscreenMode": "string(default='center')",
+        "defaultFilter": "string(default='normal')",
+        "keepMouseCentered": "boolean(default=False)",
+        "saveShortcutChanges": "boolean(default=False)",
+    }
+}
+
+# Initialize configuration automatically on import
+config.conf.spec.update(confspec)
+
```

Comment:
please see previously mentioned concerns about this, this is not safe to do here. Please add the config to configSpec